### ISSUE TITLE
Execution order label position

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#105876]
Let's use the to continue the discussion from https://github.com/microsoft/vscode/issues/104680 and Teams in September. The current location is aligned with the bottom of the cell. If we keep that location, it needs polish, for example, for overflow behavior and growing to the left.

**Comment by cpoptic** [comment]
Moving this comment here verbatim from https://github.com/microsoft/vscode/issues/104680 on request of @roblourens:

I will say @greazer has an excellent point: a key design consideration should be a very clear indicator of "Yes this cell has executed" vs "No, this cell has not yet executed".

The Execution count placement is somewhat secondary to that.

Regarding the four design options, however:

This one is not optimal for the simple reason that the cell count number is not left of the cells. DSs want to scan through a notebook at a quick glance to find "cell number x", and this Option 1 makes it much harder to find a certain cell. Not to mention some users could confuse that execution value with actual code in the cell, which we definitely don't want to do.

I really don't like how the mouse hover over causes the execution value to disappear. Veto this one.

Not bad, but I agree with other feedback that the movement of the [1] could be distracting/annoying.

I like this one except as others have pointed out, for large cells you lose the execution count value off the bottom of the screen if the cell has a bunch of lines. This is a pretty big dealbreaker, as ideally you don't want to have to scroll down just to see which cell count you're on; it adds a tiny amount of unnecessary cognitive load.

Option 5) What about simply vertically stacking both the [1] execution value and the triangle "Run cell" button? Statically, unlike Option 3. That way both the executed count value and the "Run cell" button are both always displayed, and they're both left of the cell contents, and thus less obtrusive.

So in conclusion, Option 3 (or some variant of it like my idea for Option 5) gets my vote. But I'm just a data scientist, not a designer; I'm only voicing what I (and probably other data scientists) would value in this tool.

**Comment by greazer** [comment]
Thanks for your feedback @cpoptic, as a data scientist your feedback is extremely valuable for this effort! I'll +1 your suggestion for your proposed Option 5. Based on what I've heard from other DS's it can be critical to be able to view the execution count with as little "shenanigans" required as possible.  😊

**Comment by cpoptic** [comment]
Thanks @greazer , happy to help!  Also is the addition of the "execution time" at the bottom of each notebook cell going to be added in the upcoming release, or is that from some extension?

**Comment by claudiaregio** [comment]
@cpoptic, this is currently a feature supported by the new Native Notebooks Experience. 

If you would like to try it out, details and instructions on how to do so can be found here: https://devblogs.microsoft.com/python/notebooks-are-getting-revamped/

We would love to have you try it out and let us know your thoughts! 
If you choose to do so, you can:
- Leave us a comment on GitHub **(best option!)**
- Fill out the <5 min survey that pops up
- Email claudia.regio@microsoft.com with the subject line "Native Notebooks Feedback". 

Looking forward to hearing from you!

**Comment by tanhakabir** [comment]
There hasn't been much activity in this issue. I'm assuming the current Execution Order placement is fine and will close this issue but let me know if anyone thinks otherwise!
